prboom2/src/m_random.c

```diff
@@ -146,6 +147,45 @@ void M_ClearRandom (void)
   rng.prndindex = rng.rndindex = 0;    // clear two compatibility indices
 }
 
+// [XA] Common random formulas used by codepointers
+
+//
+// P_RandomHitscanAngle
+// Outputs a random angle between (-spread, spread), as an int ('cause it can be negative).
+//   spread: Maximum angle (degrees, in fixed point -- not BAM!)
+//
+int P_RandomHitscanAngle(pr_class_t pr_class, fixed_t spread)
```

Comment:
If you're taking pr_class as an argument, you should use it for the P_Random calls, right? :^)

Reply:
Welp, brainfart. :P -- Fixed.